docs/workflow.rst

```diff
@@ -440,3 +341,69 @@ results for the stars in that region.
         --filedir spatial
 
 You may wish to use these files as inputs for the `MegaBEAST <https://megabeast.readthedocs.io/en/latest/>`_.
+
+
+**************
+Python wrapper
+**************
+
+This is a wrapper for each of the commands described above:
+
+.. code-block:: console
+
+   beast/examples/production_runs_2019/beast_production_wrapper.py
```

Comment:
Not sure this should be in a code block.  Maybe just put it in ``string`` quotes?

Reply:
done